doc/user/content/installation/_index.md

```diff
@@ -175,6 +175,29 @@ kubectl get materialize -n materialize-environment -w
 # Check the logs of the operator
 kubectl logs -l app.kubernetes.io/name=materialize-operator -n materialize
 ```
+
+### Cancelling the Upgrade
+To cancel an in-progress rollout and revert to the last completed rollout state, first retrieve the last rollout request ID from your Materialize CR. You can do this with the following command:
+
+```shell
+kubectl get materialize <instance-name> -n materialize-environment -o jsonpath='{.status.lastCompletedRolloutRequest}'
+```
+
+Next, get the previous `environmentdImageRef` from the `last-applied-configuration` annotation:
+```shell
+kubectl get materialize <instance-name> -n materialize-environment -o jsonpath='{.metadata.annotations.kubectl\.kubernetes\.io/last-applied-configuration}'
+```
+Finally, run the following command using the values obtained above:
+
+```shell
+kubectl patch materialize <instance-name> \
+  -n materialize-environment \
+  --type='merge' \
+  -p "{\"spec\": {\"requestRollout\": \"<lastCompletedRolloutRequest-value>\", \"environmentdImageRef\": \"<previous-environmentdImageRef>\" }}"
+```
```

Comment:
In general I like the idea of having examples, but I also think we need to also make sure we're providing adequate context on the interfaces we're providing. People are going to deploy this through terraform, kustomize, pulumi, and who knows what else.

I would keep what we have here, but also talk about this a bit more in the context of the Materialize spec, when in the rollout one might want to use this (and why). I also am not sure about this process. We don't support downgrade so we need to make sure we're not rolling back a completed upgrade.

Reply:
Change is here https://github.com/MaterializeInc/materialize/pull/34226/commits/02837677f6495c0660d6b0c883be29601d39c1ac. However I think I added this instruction because if you were to just reset the `requestRollout` field in `upgrade-materialize.yaml` but keep the `environmentdImageRef` you tried to update with, then it doesn't fully go back to the original state. Notably the balancerd pods don't get cleaned up: 

<img width="933" height="217" alt="Screenshot 2025-11-21 at 4 39 36 PM" src="https://github.com/user-attachments/assets/53f62718-297b-4907-867d-2665f3f49acf" />

But this might just be a bug w/ orchestratord

Reply:
oh, yeah, that's just a bug - i'm currently rewriting how balancerd gets deployed, i'll make sure to fix that as part of this